### ISSUE TITLE
brew install rvm

### ISSUE THREAD

**Issue by lifeiscontent** [rvm/rvm#3745]
any chance you guys will release a install from homebrew formula? It's one of the only packages I can't install from homebrew in my workflow.

**Comment by mpapis** [comment]
not sure if it's worth the effort right now, if you got time I would be happy to support you in the process

**Comment by lifeiscontent** [comment]
@mpapis it seems like the build process is a bit hard to follow, do you have any points of interests I should be looking at to figure out how the process to build rvm works?
